src/main/java/org/chiches/foodrootservir/dto/OrderDTO.java

```diff
@@ -1,13 +1,20 @@
 package org.chiches.foodrootservir.dto;
 
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+
 import java.time.LocalDateTime;
 import java.util.List;
 
 public class OrderDTO {
+    @NotNull(message = "Full price cannot be empty")
     private Double fullPrice;
+    @NotBlank(message = "Status cannot be empty")
     private String status;
```

Comment:
status will be empty during creation an order. it will be assigned in OrderService